### PR TITLE
Small docs updates

### LINKED ISSUE #308: Tighten terminology in docs
Quoting @haakon-e 

_It seems the terminology "class" or "subclass" is used a few locations throughout the documentation to describe parameter distributions both relating to function parameters, but also in general. I think it would be good to be a bit more precise to refer to these as abstract types and structs._

### LINKED ISSUE #311: Learning rate schedulers code snippet in documentation
It would be good to have a small code snippet of how to use the learning rate schedulers in [learning_rate_scheduler.md](https://github.com/CliMA/EnsembleKalmanProcesses.jl/blob/main/docs/src/learning_rate_scheduler.md), especially dealing with the termination condition, since this is not obvious now.

### PR DESCRIPTION
**Pull request by odunbar**
## Purpose 

Closes #311 
Closes #308 

----
- [x] I have read and checked the items on the review checklist.

### CODE REVIEW

**Comment by eviatarbach** [comment]
Looks good, but can you please add an example of how to do the EKI loop with a scheduler that terminates early?

**Comment by odunbar** [comment]
Oh I see what you are asking for now! Will do.

**Review by eviatarbach — APPROVED** [review]
LGTM!
